test/setup.sh

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
+set -aeuo pipefail
+
+UPTEST_GCP_PROJECT=${UPTEST_GCP_PROJECT:-official-provider-testing}
+
+echo "Running setup.sh"
+echo "Waiting until all configuration packages are healthy/installed..."
+"${KUBECTL}" wait configuration.pkg --all --for=condition=Healthy --timeout 5m
+"${KUBECTL}" wait configuration.pkg --all --for=condition=Installed --timeout 5m
```

Comment:
Please add configurationrevisisons check
```suggestion
"${KUBECTL}" wait configuration.pkg --all --for=condition=Installed --timeout 5m
"${KUBECTL}" wait configurationrevisions.pkg --all --for=condition=Healthy --timeout 5m
```